tests/Unit/EventListener/ExceptionListenerTest.php

```diff
@@ -25,6 +25,11 @@ public function setUp(): void {
 		$this->dispatcher = new EventDispatcher();
 	}
 
+	/** solves tests marked as "risky" by phpunit, error was: "Test code or tested code did not remove its own exception handlers" */
+	public function tearDown(): void {
+		restore_exception_handler();
+	}
+
```

Comment:
You can see how I solved this bug in Symfony for the Fundraising Application here: https://github.com/wmde/fundraising-application/pull/3019

calling `restore_exception_handler` in the `tearDown` is also totally valid, but would also need a comment why we do it and if we ever add more tests inheriting from `KernelTestCase` we'd have to place a `tearDown` method in each of them